Services/CO.CDP.Organisation.WebApi.Tests/UseCase/GetConnectedEntitiesUseCaseTest.cs

```diff
@@ -0,0 +1,67 @@
+using CO.CDP.Organisation.WebApi.Tests.AutoMapper;
+using CO.CDP.Organisation.WebApi.UseCase;
+using CO.CDP.OrganisationInformation.Persistence;
+using FluentAssertions;
+using Moq;
+
+namespace CO.CDP.Organisation.WebApi.Tests.UseCase;
+public class GetConnectedEntitiesUseCaseTest(AutoMapperFixture mapperFixture) : IClassFixture<AutoMapperFixture>
+{
+    private readonly Mock<IConnectedEntityRepository> _repository = new();
+    private GetConnectedEntitiesUseCase UseCase => new(_repository.Object, mapperFixture.Mapper);
+
+    [Fact]
+    public async Task ItReturnsEmptyIfNoConnectedEntityIsFound()
+    {
+        var organisationId = Guid.NewGuid();
+
+        var found = await UseCase.Execute(organisationId);
+
+        found.Should().BeEmpty();
+    }
+
+    [Fact]
+    public async Task ItReturnsTheFoundConnectedEntities()
+    {
+        var organisationId = Guid.NewGuid();
+        var eid1 = Guid.NewGuid();
+        var eid2 = Guid.NewGuid();
+
+        var persistenceConnectedEntityList = new List<OrganisationInformation.Persistence.ConnectedEntityLookup>
+        {
+            new OrganisationInformation.Persistence.ConnectedEntityLookup
```

Comment:
I'd just call it `ConnectedEntity` as the endpoint returns a list of connected entities.